plugins/hypervisors/kvm/src/main/java/org/apache/cloudstack/utils/qemu/QemuImg.java

```diff
@@ -30,8 +31,13 @@
 
     /* The qemu-img binary. We expect this to be in $PATH */
     public String _qemuImgPath = "qemu-img";
+    private String cloudQemuImgPath = "cloud-qemu-img";
```

Comment:
This binary does not exist on Ubuntu/Debian. Why the name change?

Reply:
Just to make sure it works with snapshot support. If the binary is present on the KVM host it will be used, and if not just the qemu-img.